CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 - Add `Walking` entity - [#88](https://github.com/IronFoundation/iOS-Woof/pull/88)
 - Add `City` property to the `owner` entity - [#93](https://github.com/IronFoundation/iOS-Woof/pull/93)
+- Add `isOwnerLoggedOnce` and `isSitterLoggedOnce` properties to struct`Preferences` - [#100](https://github.com/IronFoundation/iOS-Woof/pull/100)
```

Comment:
We added not only properties but methods as well. Do you think we need to mention that? Maybe we can express that in a more general way?
```suggestion
- Add the possibility to track if the owner/sitter was logged in at least once  - [#100](https://github.com/IronFoundation/iOS-Woof/pull/100)
```

Reply:
Fixed